pyrobosim/pyrobosim/core/world.py

```diff
@@ -1104,7 +1110,7 @@ def collides_with_robots(self, pose, robot=None):
         radius = self.inflation_radius if robot is None else robot.radius
         robot_list = [r for r in self.robots if r is not robot]
         for r in robot_list:
-            if pose.get_linear_distance(r.pose) < (radius + robot.radius):
+            if pose.get_linear_distance(r.pose) < (radius + r.radius):
                 return True
```

Comment:
```suggestion
        radius = self.inflation_radius if robot is None else robot.radius
        robot_list = [other_bot for other_bot in self.robots if other_bot is not robot]
        for other_bot in robot_list:
            if pose.get_linear_distance(other_bot.pose) < (radius + other_bot.radius):
                return True
```

Reply:
hah -- yes, I've drilled the "don't use single letter variables" thing into your head too, it seems. Will update :)

Reply:
LOL totally!!